build.gradle

```diff
@@ -19,6 +19,15 @@ repositories {
 }
 
 dependencies {
+    runtimeOnly 'org.mariadb.jdbc:mariadb-java-client'
+    runtimeOnly 'com.h2database:h2'
```

Comment:
that's.... interesting 
<img width="220" height="198" alt="image" src="https://github.com/user-attachments/assets/cfa551bb-6576-480a-b5d4-22e546585dc1" />